milestone2.ts

```diff
@@ -139,23 +146,28 @@ type UpdateVolunteer = (
   vol: Volunteer,
   updates: Partial<Volunteer>
 ) => Volunteer;
-let updateVolunteer: UpdateVolunteer; // Code here
+let updateVolunteer: UpdateVolunteer = (vol, updates) => {
+  return {...vol, ...updates};                                               
+}; 
 
-// let kyleNew = updateVolunteer(kyleClone);
-//console.log(kyleNew);
+kyleNew = updateVolunteer(kyleClone, updatedLocation);
+console.log(kyleNew);
 
 // Question 7: Object Destructuring
 /* Now that we have our updated kyle, lets use object destructuring to get his name, age, and city
  */
 
 type GetVolunteerInfo = (vol: Volunteer) => String;
 let getVolunteerInfo: GetVolunteerInfo = (vol) => {
-  let name, age, city; // Code here
+  let name, age, city;
+  name = vol.name;
+  age = vol.age;
+  city = vol.city;
```

Comment:
Looks like you aren't using object destructuring here and are instead manually assigning each value. Can you resubmit using object destructuring on this question instead? You can find an example of object destructuring in the demo from bootcamp session 3 [**here**](https://h4i.notion.site/Intro-to-Javascript-83f29480d7c14b0c9163212b96f6ab0f)